src/containers/VerifyActivities.jsx

```diff
@@ -84,7 +84,7 @@ class VerifyActivities extends Component {
                           id={activity.id}
                           category={activity.category}
                           date={(activity.date)}
```

Comment:
why are the brackets needed here?

Reply:
Rectified this. I omitted the function to format the date.